modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -1922,13 +1923,82 @@
        }
      }
    },
-   "DASHBOARD":{
+   "DASHBOARD": {
        "RUNNING":   "Running",
        "FINISHED":  "Finished",
        "PAUSED" :   "Paused",
        "FAILED" :   "Failed",
        "FINISHED_WITH_COMMENTS": "Todo",
        "SCHEDULED" : "Scheduled",
        "RECORDING" : "Recording"
+   },
+   "HOTKEYS": {
+       "CHEAT_SHEET": {
+           "TITLE": "Keyboard shortcuts"
+       },
+       "KEYS": {
+           "CTRL": "Ctrl",
+           "ALT": "Alt",
+           "OPTION": "⌥",
+           "SPACE": "␣",
+           "BACKSPACE": "⌫",
+           "LEFT": "←",
+           "RIGHT": "→",
+           "UP": "↑",
+           "DOWN": "↓",
+           "COMMAND": "⌘",
+           "SHIFT": "⇧",
+           "RETURN": "⏎",
+           "ENTER": "⏎",
+           "PLUS": "+",
+           "TAB": "↹",
+           "CAPSLOCK": "⇪",
+           "ESC": "Esc",
+           "ESCAPE": "Esc",
+           "PAGEUP": "Page↑",
+           "PAGEDOWN": "Page↓",
+           "END": "End",
+           "HOME": "Home",
+           "INS": "Ins",
+           "DEL": "Del"
+       },
+       "GROUPS": {
+           "GENERAL": "General",
+           "PLAYER": "Player",
+           "EDITOR": "Editor"
+       },
+       "DESCRIPTIONS": {
+           "GENERAL": {
+               "MAIN_MENU": "Main menu",
+               "SELECT_NEXT_DASHBOARD_FILTER": "Select next dashboard filter",
+               "SELECT_PREVIOUS_DASHBOARD_FILTER":  "Select previous dashboard filter",
+               "REMOVE_FILTERS": "Remove filters",
+               "EVENT_VIEW": "Open events table",
+               "SERIES_VIEW": "Open series table",
+               "NEW_EVENT": "Create new event",
```

Comment:
"Add event" and "Add series" would be more consistent with the labels of the buttons that do the same think.

What do you think?